api/src/api/v1/users/volunteers/__tests__/volunteer_logs.test.integration.ts

```diff
@@ -54,7 +54,7 @@ describe('API /users/me/volunteer-logs', () => {
       const res = await server.inject({
```

Comment:
How come these tests aren't using the `injectCfg` wrapper?

Reply:
I did those first before I wrote the wrapper

Reply:
Can we wrap these for consistency?